proposals/NNNN-atf-improvment-proposal.md

```diff
@@ -0,0 +1,79 @@
+# Improve Automated test framework
+
+* Proposal: [SDL-NNNN](NNNN-atf-improvement-proposal.md)
+* Author: [Alexander Kutsan](https://github.com/smartdevicelink)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / ATF]
+
+## Introduction
+
+The proposal is to make ATF more flexible, salable and easy to use and integrate with popular CI systems and to analyze and full of all required information. 
+Also some functionality of ATF should be moved to scripts common files due to it's dependency from SDL version and features.
+
+## Motivation
+
+Current ATF requires and unfortunately allows changes during testing specific SDL scenarios. 
+Also it requires additional environment preparation and management during testing test sets
```

Comment:
It also requires additional environment preparation/configuration and control during performing test sets